### PR TITLE
Docker multi-stage maven builder

### PR DESCRIPTION
**Pull request by Johnny850807**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM maven:3.8.4-jdk-11-slim as builder
+
+COPY common /usr/src/app/common
+COPY spring-boot-dixit /usr/src/app/spring-boot-dixit
+COPY dixit /usr/src/app/dixit
+COPY pom.xml /usr/src/app
+
+RUN mvn -f /usr/src/app/pom.xml clean package
```
Comment (wallychen0705) [inline Dockerfile:8]: NIT: Can't it be skipped the test phrase?

**Review by wallychen0705 — COMMENTED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (Johnny850807) [reply Dockerfile:8]: good advice

**Review by wallychen0705 — APPROVED** [review]
LGTM

**Pull request merged**